packages/langium/src/lsp/completion/completion-provider.ts

```diff
@@ -296,6 +309,12 @@ export class DefaultCompletionProvider implements CompletionProvider {
         }
     }
 
+    protected performNextTokenCompletion(document: LangiumDocument, text: string, _offset: number, _end: number): boolean {
+        // This regex returns false if the text ends with a letter
+        // We don't want to complete new text immediately after a keyword, ID etc.
+        return /\P{L}$/u.test(text);
```

Comment:
Please add a comment on the meaning of the regEx, and motivate why not `/\w$/`,
and that the pattern is expected to care only for the last character within `text` (although it's implicitly mentioned within the first comment line)